source/common/umf_pools/disjoint_pool_config_parser.hpp

```diff
@@ -12,18 +12,36 @@
 #include <umf/pools/pool_disjoint.h>
 
 #include <memory>
+#include <mutex>
 #include <string>
+#include <thread>
```

Comment:
Why do we need these includes?

Reply:
Good catch, removed. It was a leftover from previous implementation.